components/schemes/Color/src/MDCSemanticColorScheme.h

```diff
@@ -116,4 +116,8 @@ typedef NS_ENUM(NSInteger, MDCColorSchemeDefaults) {
  */
 - (nonnull instancetype)initWithDefaults:(MDCColorSchemeDefaults)defaults;
 
+/** RGBA to RGB conversion */
```

Comment:
Naming is hard.
blendColor:withBackgroundColor: may be more specific.
Please give a detailed description on how / why the two colors are merged.
(RGBA -> RGB is super detailed)